pkg/cli/environment.go

```diff
@@ -41,11 +43,8 @@ const defaultMaxHistory = 10
 // EnvSettings is a composite type of helm.pkg.env.EnvSettings.
 // describes all of the environment settings.
 type EnvSettings struct {
+	EnvSettings *cli.EnvSettings
```

Comment:
I am not sure if this will still be handled as composition or just as child element. As far as I remember the second will happen....

Reply:
I think so, the second, because the fields are not propagated into our settings struct anymore

Reply:
From that point of view this has become correct yes :)